vscode/microsoft-kiota/src/steps.ts

```diff
@@ -240,7 +255,71 @@ export async function generateSteps(existingConfiguration: Partial<GenerateState
 		});
 		state.language = pick.label.split('-')[0].trim();
 	}
-    await MultiStepInput.run(input => inputClientClassName(input, state), () => step-=2);
+    async function inputPluginName(input:MultiStepInput, state: Partial<GenerateState>) {
+        state.pluginName = await input.showInputBox({
+            title: `${l10n.t('Create a new plugin')} - ${('plugin name')}`,
+            step: step++,
+            totalSteps: 3,
+            value: state.pluginName || '',
```

Comment:
```suggestion
            value: state.pluginName ?? '',
```
Nit: with nullish and OR operators. Throughout similar code sections in `steps.ts` where a `Partial<T>` is used to make type T values optional values, when you do a `T.property`, `property` can either be defined or undefined. Therefore, using a nullish operator should be preferred over the OR which evaluates over `falsy` values (0, "").

Reply:
Okay, yeah I see what you're saying! I'll update this